info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -76,6 +82,38 @@ public InfoCompiler(Datastore datastore) throws IOException {
     this.datastore = datastore;
     this.webCrawler =
         new WebCrawler(this.datastore, new NewsContentExtractor(), new RelevancyChecker());
+    parseAddressesFromDataset();
+  }
+
+  /**
+   * Reads the National Address Database (Release 3) shown here @link <a href=
+   * "https://www.transportation.gov/gis/national-address-database/national-address-database-0">
+   * reference</a>, from Google Cloud Storage. The file is located at bucket {@code
+   * Config.ADDRESS_BUCKET_NAME} and named {@code Config.ADDRESS_FILE_NAME}.
+   */
```

Comment:
```suggestion
  /**
   * Reads the National Address Database (Release 3), referenced below, from Google Cloud Storage. The file is located at bucket {@code
   * Config.ADDRESS_BUCKET_NAME} and named {@code Config.ADDRESS_FILE_NAME}.
   *
   * @see <a href="https://www.transportation.gov/gis/national-address-database/national-address-database-0">National Address Database (Release 3)</a>
   */
```

Reply:
Spacing might be off, but I think this is the general idea:
https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see

Reply:
Thanks!